adapters/feeder2core/feeder2core.go

```diff
@@ -218,7 +219,7 @@ func AdaptDeployAccountTransaction(t *feeder.Transaction) *core.DeployAccountTra
 	}
 }
 
-func AdaptCairo1Class(response *feeder.SierraDefinition, compiledClass json.RawMessage) (core.Class, error) {
+func AdaptCairo1Class(response *feeder.SierraDefinition, compiledClass *feeder.CompiledClass) (core.Class, error) {
```

Comment:
This can return *core.Cairo1Class. Let's make a separate adapter `AdaptCompiledClass`.

Reply:
```go
func AdaptCairo1Class(*feeder.SierraDefinition) (*core.Cairo1Class, error)

func AdaptCompiledClass(*feeder.CompiledClass) (*core.CompiledClass, error)
```
We can do something like this^